packages/url_launcher/url_launcher/README.md

```diff
@@ -8,6 +8,22 @@ iOS, Android, web, Windows, macOS, and Linux.
 ## Usage
 To use this plugin, add `url_launcher` as a [dependency in your pubspec.yaml file](https://flutter.dev/platform-plugins/).
 
+## Installation
+
+### iOS 
+
+Add the following keys to your _Info.plist_ file, located in `<project root>/ios/Runner/Info.plist`:
```

Comment:
This is not an accurate summary of the requirement, and may still cause confusion. The correct description is that the scheme of any URL they pass to `canLaunch` should be listed, and the README should say that. You can give the below as an *example*, but you should make clear that it's an example.

Reply:
Made the changes,  thanks!